### PR TITLE
More TodoList upgrades

### PR DESCRIPTION
**Pull request by rgora23**
- Added new styling to allow for centered todo list.
- Updated styling for delete button with new hover color

Todo

### CODE REVIEW

File: src/pages/TodoList.tsx [hunk 1]
```diff
@@ -8,11 +8,36 @@ const TodoListItem = styled.div`
...
+    padding: 5px;
+    margin: 2em 2em 2em 0em;
+    border-radius: 12px;
+    background-color: #fce25b;
+
+    &:hover {
+      font-weight: 900;
+      color: red;
+    }
+  }
```
Comment (lim-is-a) [inline src/pages/TodoList.tsx:26]: For the button styling I prefer to keep the current color, so the app looks more streamlined. 
If what you are looking for is something to differentiate the button for delete, changing the hover styling will suffice.

File: src/pages/TodoList.tsx [hunk 2]
```diff
@@ -8,11 +8,36 @@ const TodoListItem = styled.div`
...
+`
+
+const TodoListBody = styled.div`
+  display: flex;
+  justify-content: center;
+  max-width: 50%;
+  overflow: text-wrap;
```
Comment (lim-is-a) [inline src/pages/TodoList.tsx:33]: The center todolist styling is a nice app update, thank you.

**Review by lim-is-a — CHANGES_REQUESTED** [review]

File: src/pages/TodoList.tsx (hunk 1 above)
Comment (rgora23) [reply src/pages/TodoList.tsx:26]: I understand, Yeah the color change to yellow wasn't really the best looking change here.

Comment (lim-is-a) [reply src/pages/TodoList.tsx:25]: Thank you for making the changes.

**Review by lim-is-a — APPROVED** [review]
Looks good, thank you for making the changes.

**Pull request merged**